Library/Formula/git-hooks.rb

```diff
@@ -0,0 +1,28 @@
+class GitHooks < Formula
+  homepage "https://github.com/icefox/git-hooks"
+  url "https://github.com/icefox/git-hooks/archive/1.00.0.tar.gz"
+  version "1.00.0"
```

Comment:
`brew audit --strict` doesn't like this. The version is already scanned from the `url`.